Backpack/src/main/java/net/skyscanner/backpack/calendar/presenter/BpkCalendarController.kt

```diff
@@ -70,7 +73,14 @@ abstract class BpkCalendarController {
   }
 
   private companion object {
-    val DEFAULT_START_DATE: Calendar = Calendar.getInstance()
-    val DEFAULT_END_DATE: Calendar = Calendar.getInstance().apply { add(Calendar.YEAR, 1) }
+    val DEFAULT_START_DATE: CalendarDay = Calendar.getInstance().toCalendarDay()
+    val DEFAULT_END_DATE: CalendarDay = Calendar.getInstance()
+      .apply {
+        add(Calendar.YEAR, 1)
+        add(Calendar.DATE, -1)
+      }
+      .toCalendarDay()
   }
 }
+
+fun Calendar.toCalendarDay() = CalendarDay(year = get(Calendar.YEAR), month = get(Calendar.MONTH), day = get(Calendar.DAY_OF_MONTH))
```

Comment:
this can be an additional constructor in `CalendarDay` class as 

``` 
constructor(day: Calendar) : this(day.get(Calendar.YEAR), day.get(Calendar.MONTH), day.get(Calendar.DAY_OF_MONTH))
```
as it is the responsibility of that class and is more discoverable to the consumer.

Reply:
Calendar is a bad, mutable api that I intentionally removed from the CalendarDay api to keep it clean. I moved it to call site so it's easier to be removed.

Reply:
Makes sense. Can we make `toCalendarDay` internal to make sure is not used outside our lib?

Reply:
👍